tests/layer_tests/tensorflow_tests/test_tf_AddN.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+
+from common.tf_layer_test_class import CommonTFLayerTest
+
+import logging
+
+# Testing operation AddN
+# Documentation: https://www.tensorflow.org/api_docs/python/tf/raw_ops/AddN
+
+class TestAddN(CommonTFLayerTest):
+    # Overload inputs generation to fill dummy input
+    def _prepare_input(self, inputs_dict):
+        for input in inputs_dict.keys():
+            inputs_dict[input] = np.random.randint(-3, 3, inputs_dict[input]).astype(np.float32)
+        return inputs_dict
+    # input_list - should be an array, could be a single shape, or array of n-dimentional shapes
+    # ir_version - common parameter
+    # use_new_frontend - common parameter
+    def create_addn_placeholder_const_net(self, input_list, ir_version, use_new_frontend):
+        """
+            Tensorflow net                  IR net
+
+            Placeholder->AddN      =>       Placeholder->AddN
+                         /                               /
+            Const-------/                   Const-------/
+
+        """
+
+        if len(input_list) == 0:
+            raise RuntimeError("Input list couldn't be empty")
+        import tensorflow as tf
+
+        tf.compat.v1.reset_default_graph()
+
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            tf_inputs = []
+
+            for item in input_list:
+                tf_shape = item.copy()
```

Comment:
not needed to copy it. Please rename `item` -> `input_shape` and `input_list` -> `input_shapes`

Reply:
done